_layouts/meltano_plugin.html

```diff
@@ -9,6 +9,44 @@
 {% endcapture %}
 {% if plugin.metrics %} {% assign metrics = plugin.metrics %}{% endif %}
 
+<!-- To determine if the current plugin is pure Singer or not -->
+{% assign is_singer = false %}
```

Comment:
We may be better off moving this logic to https://github.com/meltano/hub/blob/main/_plugins/meltano_plugin_page_generator.rb so that we can also use it to toggle the page title and header (https://github.com/meltano/hub/issues/548).

If we add it to `@data` there, it becomes available as `page.is_singer` here, and likely in `_includes/setting.html` as well -- although we may still need to pass it there explicitly.

Reply:
@DouweM I was able to move the logic and update everything. Much easier to do it in the Ruby script!

I'm going to open another PR targeting this branch that will work to fix the other plugin pages. I've got a few commits ready but will have to wait until tomorrow to continue it.